lib/galaxy/managers/hdas.py

```diff
@@ -580,8 +580,9 @@ def serialize_copied_from_ldda_id(self, item, key, **context):
         """
         Serialize an id attribute of `item`.
         """
-        if item.copied_from_library_dataset_dataset_association is not None:
-            return self.app.security.encode_id(item.copied_from_library_dataset_dataset_association.id)
+        copied_from_ldda_id = item.copied_from_library_dataset_dataset_association
```

Comment:
That seems a little awkward, you can use `item.copied_from_library_dataset_dataset_association_id` directly